OloEngine/src/OloEngine/Audio/SoundGraph/NodeSchema.cpp

```
@@ -0,0 +1,290 @@
+    } // namespace
+
+    const NodePinSchema* GetNodePinSchema(const std::string& nodeType)
+    {
+        const auto& map = GetPinSchemaMap();
+        auto it = map.find(nodeType);
+        if (it == map.end())
+            return nullptr;
+        return &it->second;
+    }
+
+    f32 ParsePropertyFloat(const NodeParamSchema& schema, const std::string& valueStr)
```

Cppcheck (2.20.0)

[style] The function 'ParsePropertyFloat' is never used.

(unusedFunction)